docs/ossm/cert-manager/README.md

```diff
@@ -0,0 +1,200 @@
+[Return to OSSM Docs](../)
+
+# Cert Manager and istio-csr Integration
+
+Below are instructions for integrating cert-manager with OpenShift Service Mesh 3. It largely follows the [cert-manager istio-csr documentation](https://cert-manager.io/docs/usage/istio-csr/) but you will need to adjust a few settings based on which `updateStrategy` you are using.
+
+## Common Setup
+
+These steps are the same for each `updateStrategy`.
+
+1. Install the [cert-manager Operator for Red Hat OpenShift](https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/security_and_compliance/cert-manager-operator-for-red-hat-openshift#cert-manager-operator-install).
+
+2. Create the `istio-system` namespace for the root cert and for your `Istio` resource.
+
+   ```sh
+   oc create namespace istio-system
+   ```
+
+3. Create a Self Signed certificate. Note you should adapt this example based on what PKI you are using e.g. using a vault `Issuer` instead of a self signed `Issuer`.
+
+   ```sh
+   oc apply -f https://raw.githubusercontent.com/cert-manager/website/7f5b2be9dd67831574b9bde2407bed4a920b691c/content/docs/tutorials/istio-csr/example/example-issuer.yaml
+   ```
+
+4. Export the root CA to the `cert-manager` namespace.
+
+   ```sh
+   oc get -n istio-system secret istio-ca -o jsonpath='{.data.tls\.crt}' | base64 -d > ca.pem
+   oc create secret generic -n cert-manager istio-root-ca --from-file=ca.pem=ca.pem
+   ```
+
+## InPlace Strategy
+
+1. Install [istio-csr](https://cert-manager.io/docs/usage/istio-csr).
+
+   ```sh
+   helm repo add jetstack https://charts.jetstack.io --force-update
+
+   helm upgrade cert-manager-istio-csr jetstack/cert-manager-istio-csr \
+     --install \
+     --namespace cert-manager \
+     --wait \
+     --set "app.tls.rootCAFile=/var/run/secrets/istio-csr/ca.pem" \
+     --set "volumeMounts[0].name=root-ca" \
+     --set "volumeMounts[0].mountPath=/var/run/secrets/istio-csr" \
+     --set "volumes[0].name=root-ca" \
+     --set "volumes[0].secret.secretName=istio-root-ca" \
+     --set "app.istio.namespace=istio-system"
+   ```
+
+   Note: If your controlplane namespace is not `istio-system`, you will need to update `app.istio.namespace` to match your controlplane namespace.
+
+2. Install `Istio` controlplane in the `istio-system` namespace.
+
+   ```sh
+   oc apply -f - <<EOF
+   apiVersion: sailoperator.io/v1alpha1
+   kind: Istio
+   metadata:
+     name: default
+   spec:
+     version: v1.23.0
+     namespace: istio-system
+     updateStrategy:
+       type: InPlace
+     values:
+       global:
+         caAddress: cert-manager-istio-csr.cert-manager.svc:443
+       pilot:
+         env:
+           ENABLE_CA_SERVER: "false"
+         volumeMounts:
+           - mountPath: /tmp/var/run/secrets/istiod/tls
```

Comment:
JFYI: We do not customize this location in OSSM 2.x and community Istio does not need it as well.

Reply:
This is necessary because the OSSM pilot image sets the working dir to `/tmp`. I don't think this is done explicitly but rather set by some base image. There's an open issue to address this: https://issues.redhat.com/browse/OSSM-8194